inc/Addon/WebP/Subscriber.php

```diff
@@ -435,14 +435,20 @@ private function url_to_webp( $url, $extensions ) {
 
 		if ( $this->filesystem->exists( $src_path_webp ) ) {
 			// File name: image.jpg => image.webp.
-			return preg_replace( '@\.' . $src_url['extension'] . '$@', '.webp', $src_url['src'] ) . $src_url['query'];
+			$src_url_webp = preg_replace( '@\.' . $src_url['extension'] . '$@', '.webp', $src_url['src'] ) . $src_url['query'];
+		} elseif ( $this->filesystem->exists( $src_path_webp = $src_path . '.webp' ) ) { // phpcs:ignore Generic.Sniffs.CodeAnalysis.AssignmentInCondition,Squiz.Sniffs.PHP.DisallowMultipleAssignments
```

Comment:
Assigning within in the condition avoids unnecessarily nested `if` statements. it's not meant to be a comparison.